README.md

```diff
@@ -19,7 +19,8 @@ All meetings in person or online are held under [OSS Review Toolkit's charter](C
 You can find the meeting minutes in the [minutes directory](./minutes/) in this repository.
 
 Unless otherwise indicated the below names in the minutes refer to the following persons:
-- *Frank*: Frank Viernau ([@fviernau](https://github.com/fviernau)])
+- *Frank*: Frank Viernau ([@fviernau](https://github.com/fviernau))
```

Comment:
Ideally this fix should go to a separate commit.

Reply:
Tired of waiting for @tsteenbe to make this fix, so let's get this merged as I need to refer to the current TSC members 😉